MidasCivil_Adapter/CRUD/Read/Results/ReadResults.cs

```diff
@@ -203,6 +227,24 @@ private static string GetCSVFile(string path)
 
         /***************************************************/
 
+        private List<string> CaseNamesAPI(List<string> names, List<string> loadCases, List<string> loadComb)
```

Comment:
```suggestion
        private List<string> AppendCaseType(List<string> names, List<string> loadCases, List<string> loadComb)
```

I would put this in the PrivateHelpers section so it can be accessed by other methods.